spec/support/importer.rb

```diff
@@ -17,7 +17,7 @@
 shared_examples 'an importer which indexes the correct documents' do
   it 'indexes the correct documents' do
     expect(importer.model_class).to receive(:index) do |indexed_docs|
-      expect(indexed_docs).to eql(expected)
+      expect(indexed_docs).to match_array(expected)
```

Comment:
I don't think the order is relevant when checking if the correct documents got indexed, so I've changed this one.
Working around the order difference from the importer and the expected results seems like unnecessary noise.

Are my assumptions correct?